python/lsst/pipe/analysis/fakesAnalysis.py

```diff
@@ -1143,7 +1234,7 @@ def fakesCompletenessPlot(inputFakes, inputFakesMatched, processedFakesMatched,
     axRight = axLeft.twinx()
     axLeft.tick_params(axis="y", labelcolor="C0")
     axLeft.set_ylabel("Fraction Recovered", color="C0")
-    axLeft.set_xlabel("Magnitude / mags")
+    axLeft.set_xlabel("Magnitude (mags)")
```

Comment:
"Magnitude" should be replaced with something that describes the quantity like `Input PSF-Mag` or  `band + "magVar"`  Currently, it reads like you're stating the units twice.